pygments/lexers/matlab.py

```diff
@@ -72,6 +72,8 @@ class MatlabLexer(RegexLexer):
              "hilb", "invhilb", "magic", "pascal", "rosser", "toeplitz", "vander",
              "wilkinson")
 
+    _operators = r'-|==|~=|<|>|<=|>=|&&|&|~|\|\|?|\.\*|\*|\+|\.\^|\.\\|\.\/|\/|\\'
```

Comment:
this will find `<` before `<=`, the order has to be different (I know you just moved the definition)

Or, better, use `words()` which will automatically escape and order the regex to avoid this problem.

Reply:
I just reordered the elements, will pass on the bigger refactoring.